internal/server/grpc/grpc.go

```diff
@@ -179,3 +207,68 @@ func loadX509KeyPair(certfile, keyfile string) (tls.Certificate, error) {
 	}
 	return tls.X509KeyPair(cert, key)
 }
+
+func generateCertificatesForATLS(svc *agent.Service) ([]byte, []byte, error) {
```

Comment:
I don't think we need the agent service here since, I would say to pass the attestation report as bytes one generated on agent/main.go

Reply:
The problem is that we need to insert the hash of the public key of the x.509 certificate used for TLS inside the attestation report.... Maybe we can also have the method for fetching the attestation report inside the server. How does this sound? I know that we will then have duplicate code for AR, but it seem to me to be naturan this way.

Reply:
I just mean to call the attestation method in main.go once service is initialized and pass the output(attestation report []byte) to this function rather than passing the while service